db/models.py

```diff
@@ -19,34 +29,78 @@ def __str__(self) -> str:
 class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
-    actors = models.ManyToManyField(to=Actor, related_name="movies")
-    genres = models.ManyToManyField(to=Genre, related_name="movies")
+    genres = models.ManyToManyField(Genre)
+    actors = models.ManyToManyField(Actor)
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(null=True)
+    user = models.ForeignKey(get_user_model(), on_delete=models.CASCADE)
 
     def __str__(self) -> str:
-        return self.title
+        return str(self.created_at)
+
+    class Meta:
+        ordering = ["-created_at"]
+
+
+def create_order(tickets: Any, username: Any,
+                 date: Any = None) -> None:
+    with transaction.atomic():
+        user, _ = User.objects.get_or_create(username=username)
```

Comment:
Code Style: In the `create_order` function, you are creating a User object directly. It's recommended to use `get_user_model().objects.get_or_create(username=username)` instead of `User.objects.get_or_create(username=username)`.